modules/vba_documents/app/workers/vba_documents/upload_processor.rb

```diff
@@ -107,9 +107,9 @@ def validate_metadata(metadata_input)
         raise VBADocuments::UploadError.new(code: 'DOC102',
                                             detail: 'Non-numeric or invalid-length fileNumber')
       end
-    rescue JSON::ParserError
+    rescue JSON::ParserError, NoMethodError
```

Comment:
is the `NoMethodError` to catch if `metadata` isn't a hash?
if that's the case, what do you think about just testing for that after the parse?
`.is_a? Hash`
or even test that is reponds to `:keys` or `:[]`

Reply:
I guess my thought is that I wouldn't want catching NoMethodError to obscure some other problem (but I'm still just guessing at the intention)

Reply:
ya I like that approach much better, let me refactor some things one sec.